deployments/cloudfoundry/tile/tile.yml

```
@@ -0,0 +1,132 @@
+---
+# The high-level description of the tile.
+#
+name: splunk-otel-collector # By convention lowercase with dashes
+icon_file: resources/icon.png
+label: Splunk OpenTelemetry Collector for Tanzu Network
+description: |
+  This tile contains a BOSH Release of the Splunk distribution of OpenTelemetry Collector,
+  and a firehose nozzle job that sends metrics to the Splunk Observability Cloud.
+
+  The firehose nozzle pulls events off of the Loggregator Firehose via the RLP
+  Gateway and sends them to the Splunk Observability Cloud. This uses the
+  `cloudfoundry` receiver from the Splunk Opentelemetry Collector under the
+  hood.
+
+  This also includes a BOSH release of the Splunk Collector, called "splunk-otel-collector"
+  that can be used in deployments to put the collector on individual VMs.
+
+# Specify the packages to be included in the tile.
+packages:
+  - name: splunk_collector_firehose_nozzle
+    type: bosh-release
+    path: resources/latest-release.tgz
+    jobs:
+      - name: splunk-otel-collector
+        instances: 1
+        templates:
+          - name: splunk-otel-collector
+            release: splunk-otel-collector
+        properties:
+          cloudfoundry:
+            rlp_gateway:
+              endpoint: https://log-stream.(( ..cf.cloud_controller.system_domain.value ))
+              shard_id: (( .properties.cloudfoundry_rlp_gateway_shard_id.value ))
+              tls:
+                insecure_skip_verify: (( .properties.cloudfoundry_rlp_gateway_tls_insecure_skip_verify.value ))
+            uaa:
+              endpoint: https://uaa.(( ..cf.cloud_controller.system_domain.value ))
+              password: (( .properties.cloudfoundry_uaa_password.value ))
+              username: (( .properties.cloudfoundry_uaa_username.value ))
+              tls:
+                insecure_skip_verify: (( .properties.cloudfoundry_uaa_tls_insecure_skip_verify.value ))
+          otel:
+            proxy:
+              http: (( .properties.otel_proxy_http.value ))
+              https: (( .properties.otel_proxy_https.value ))
+              no: (( .properties.otel_proxy_no.value ))
+          splunk:
+            access_token: (( .properties.splunk_access_token.value ))
+            api_url: (( .properties.splunk_api_url.value ))
+            ingest_url: (( .properties.splunk_ingest_url.value ))
+            realm: (( .properties.splunk_realm.value ))
+
+# Forms allow users to input variable values for Splunk Collector Bosh Release and nozzle job
+forms:
+- name: nozzle_form
+  label: Nozzle Config
+  description: Splunk collector nozzle configuration values
```

```suggestion
  description: Splunk OpenTelemetry collector nozzle configuration values
```
Yes, I think having the consistent naming makes sense if we're pushing this out to a third party website.

---

Fixed